mm2src/coins/lp_coins.rs

```diff
@@ -4687,11 +4687,21 @@ pub trait IguanaBalanceOps {
     async fn iguana_balances(&self) -> BalanceResult<Self::BalanceObject>;
 }
 
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+/// Information about the UTXO protocol used by a coin.
+pub struct UtxoProtocolInfo {
+    /// A CAIP-2 compliant chain ID. Starts with `b122:`
+    /// This is used to identify the blockchain when using WalletConnect.
+    /// https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-4.md
+    chain_id: String,
+}
+
```

Comment:
We should start adding this to coins configs and docs @mariocynicys please open an issue in coins repo and in docs repo. c.c. @cipig @smk762